CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(SONATA_SRC
     src/hdf5_mutex.cpp
     src/nodes.cpp
     src/population.cpp
+    src/output.cpp
```

Comment:
I guess @alkino chose this name because of the title on the SONATA specification
https://github.com/alkino/sonata/blob/fix_index_count/docs/SONATA_DEVELOPER_GUIDE.md#output-file-formats

But I agree with you that report_reader(s) makes more sense